### PR TITLE
Fit to screen Checkbox for preview NFT

### PR DESCRIPTION
**Pull request by faizhashmiRnS**
## Description
Following are the issues that are resolved in this PR: 
1. Pylons-tech/pylons#925 
2. Pylons-tech/pylons#924

### CODE REVIEW

**Comment by faizhashmiRnS** [comment]
Hi @kjawadDeveloper2 , Can you please give it a review and forward it for testing?

File: easel/lib/screens/preview_screen.dart [hunk 1]
```diff
@@ -31,10 +32,40 @@ class PreviewScreen extends StatefulWidget {
...
+  Widget buildCheckBox(EaselProvider easelProvider) {
+    switch (easelProvider.nftFormat.format) {
+      case NFTTypes.image:
+        return SizedBox(
+          width: isTablet ? 0.3.sw : 0.5.sw,
+          child: Theme(
+            data: Theme.of(context).copyWith(unselectedWidgetColor: Colors.white),
+            child: CheckboxListTile(
+              dense: true,
+              // contentPadding: EdgeInsets.symmetric(horizontal: isTablet ? 50.w : 40.w),
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/preview_screen.dart:44]: Why there is dead code?

File: easel/lib/widgets/image_widget.dart [hunk 2]
```diff
@@ -11,31 +13,47 @@ class ImageWidget extends StatelessWidget {
...
+      width: 1.sw,
+      height: 1.sh,
+      fit: easelProvider.imageCheckBox ? BoxFit.fitHeight : BoxFit.fitWidth,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return WillPopScope(
+        child: Stack(
```
Comment (kjawadDeveloper2) [inline easel/lib/widgets/image_widget.dart:49]: if we remove stack we dont have to give height and width.

**Review by kjawadDeveloper2 — COMMENTED** [review]
Need to run this on android, ios and tablet to confirm.

**Comment by kjawadDeveloper2** [comment]
WhatsApp Image 2022-09-01 at 8 19 43 PM (1)

WhatsApp Image 2022-09-01 at 8 19 43 PM

@StefanB90 is the UI correct?

File: easel/lib/screens/preview_screen.dart (hunk 1 above)
Comment (faizhashmiRnS) [reply easel/lib/screens/preview_screen.dart:44]: Updated.

File: easel/lib/widgets/image_widget.dart (hunk 2 above)
Comment (faizhashmiRnS) [reply easel/lib/widgets/image_widget.dart:49]: Removed the stack, It was for the checkbox but checkbox was shifted to preview_screen but the stack was still here.

**Comment by xinyapylons** [comment]
@kjawadDeveloper2
Does this design include what you need? [https://www.figma.com/file/fhWmRya8H5Baasjq4XA35r/Easel?node-id=3933%3A509](url)

**Review by faddat — APPROVED** [review]

**Comment by StefanB90** [comment]
@faizhashmiRnS please demo this to @kjawadDeveloper2 & I when you are able to. Thank you.

**Comment by faddat** [comment]
@depoendabot recreate

**Comment by Tanyaml** [comment]
@kjawadDeveloper2 @faizhashmiRnS 
The Fit to screen Checkbox for preview NFT is not available- testing with the new build that was given on Friday. (https://we.tl/t-J0VfVTXRIi)

Please see video:

https://user-images.githubusercontent.com/93543576/190860335-84ec1384-7746-4ad1-8f4f-55a6a4ff0bdd.mov

**Comment by faizhashmiRnS** [comment]
Hi @Tanyaml , Can you please verify the that you are testing the same build that I had shared with you on Friday? 
This is the same build that I am using and we can see the checkbox.

https://user-images.githubusercontent.com/106378847/190968707-68f91854-73e2-45aa-b056-cc5ea8012302.mp4

**Comment by Tanyaml** [comment]
I will be closing this ticket since main issue here is BD and will assign this ticket to Fasial.

**Comment by Tanyaml** [comment]
It is working as expected now- Fit to screen checkbox is working. Although the color of image looks dark once user uploads the image and thats not what it look like originally. Therefore I will be closing this ticket and opening a bug for the Color Aspect issue.

**Comment by kjawadDeveloper2** [comment]
you closed this but did you confirmed with the designer? The design is i guess different.

**Comment by kjawadDeveloper2** [comment]
Secondly if the user selects fit to screen will this be only on this screen or all the screens?

**Comment by Tanyaml** [comment]
@StefanB90 can you please explain what is the expected results/Acceptance Criteria  and add Figma file for UI cross reference 

@kjawadDeveloper2 once I get the details I will retest this ticket.

**Comment by Tanyaml** [comment]
Per Figma File- the UI Design is total different for actual results.
Please see screenshots:
Screen Shot 2022-09-19 at 5 55 55 PM

Screen Shot 2022-09-19 at 5 56 01 PM

**Comment by StefanB90** [comment]
these are the reference images provided to QA that should be used for testing.

https://user-images.githubusercontent.com/93543576/191126693-dd913a6a-f999-4179-86fe-716566a926a8.png

https://user-images.githubusercontent.com/93543576/191126697-ef529d8b-1452-4486-be5f-7aed52311f56.png

**Comment by faizhashmiRnS** [comment]
Hi @StefanB90, The updated design is in progress.
This is what I wanted to have a call with you but we could not make that. Can we do that tomorrow after SCRUM?

**Review by faddat — APPROVED** [review]

**Comment by Tanyaml** [comment]
This is still not working as expected.

Please see video:


https://user-images.githubusercontent.com/93543576/191990924-c0f25006-5551-4f22-a686-767a1b8c3598.mov

**Pull request closed without merging**